concourse/steps/update_component_deps.py

```diff
@@ -410,6 +412,35 @@ def create_upgrade_pr(
     )
     from_component = from_component_descriptor.component
 
+    to_component_descriptor = component_descriptor_lookup(
+        ocm.ComponentIdentity(
+            name=to_ref.componentName,
+            version=to_version,
+        )
+    )
+
+    to_component = to_component_descriptor.component
+
+    bom_diff = cnudie.retrieve.component_diff(
+        from_component,
+        to_component
+    )
+
+    if delivery_dashboard_url:
+        delivery_dashboard_url_view_diff = (
+            f'{delivery_dashboard_url}/#/component?name={to_component.name}&view=diff'
+            f'&componentDiff={from_component.name}:{from_ref.version}'
```

Comment:
nit: Although it is not wrong, rather use consistently either `from_component` or `from_ref` here.